charts/ccsm-helm/README.md

```diff
@@ -0,0 +1,175 @@
+[![Community Extension](https://img.shields.io/badge/Community%20Extension-An%20open%20source%20community%20maintained%20project-FF4700)](https://github.com/camunda-community-hub/community)[![Lifecycle: Incubating](https://img.shields.io/badge/Lifecycle-Incubating-blue)](https://github.com/Camunda-Community-Hub/community/blob/main/extension-lifecycle.md#incubating-)[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+
+# Camunda Cloud Self-Managed Helm Chart
+
+This functionality is in beta and is subject to change. The design and code is less mature than official GA features and is provided as-is with no warranties. Beta features are not subject to the support SLA of official GA features.
+
+## Requirements
+
+* [Helm](https://helm.sh/) >= 3.x +
+* Kubernetes >= 1.20+
+* Minimum cluster requirements include the following to run this chart with default settings. All of these settings are configurable.
+  * Three Kubernetes nodes to respect the default "hard" affinity settings
+  * 2GB of RAM for the JVM heap
+
+
+## Installing
+
+* Add the official CCSM helm charts repo
+
+  ```shell
+  helm repo add ccsm https://helm.camunda.io
+  ```
+
+* Install it
+
+  ```shell
+  helm install camunda-cloud ccsm/ccsm-helm
```

Comment:
```suggestion
  helm install camunda-cloud camunda-cloud/ccsm-helm
```

Not sure about the name of the installation, would the user also call it `camunda-cloud`, it seems that is what elastic is [recommending](https://artifacthub.io/packages/helm/elastic/elasticsearch#installing)
```
helm install elasticsearch --version <version> elastic/elasticsearch
```